### ISSUE TITLE
Reviewer 2 Comment 1

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#26]
@mrshirts @jpotoff @msoroush 

> section 3.4: this case study ignores a paper where a Mie force field (also lamda=16) was earlier proposed for cyclohexane, with (I guess) essentially the same results. Differences should appear only because the objective function is slightly different here. The conclusion of this work is that the here proposed force field is „the most accurate“. Omitting the previous study is (at least) awkward, because the authors seem to be aware of the line of developments from these authors, as the reference list shows. Is there a reason for omitting the study mentioned above from the list of „most reliable force fields from literature. Ref 22,24,27,55,56,57“ ?

The reviewer never clarifies what paper has a Mie force field for cyclohexane. @jpotoff @msoroush Any ideas which paper the reviewer is referring to? Who are "these authors" that are included in our reference list? I can't find one for TAMie (the only other Mie family we cite). I know there are parameters obtained with SAFT-gamma that use fractional lambda values and some single-cite cyclohexane Mie models. But I don't think we have any data for those models.

**Comment by ramess101** [comment]
@jpotoff @msoroush 

OK, I am pretty confident that the reviewer is referring to TAMie. I didn't notice that there were some TAMie parameters reported for cyclohexane in the aldehydes and ketones paper. They did not reparameterize cyclohexane in the "individualized" study so I assumed they didn't have any parameters. 

Their parameters are quite similar to ours. I wish I would have known this beforehand, it sort of steals our thunder. 

TAMie: epsilon = 69.568 sigma = 3.8967 lambda = 16

MiPPE: epsilon = 69.7 sigma = 3.902 lambda = 16

Of course, it is still impressive that our first iteration (starting with TraPPE) yields similar parameters to the TAMie optimal:

First iteration: epsilon = 70 sigma = 3.89 lambda = 16

I could include that TAMie point on our heat map to emphasize this point.

TAMie reported tabulated property values too, so I can certainly include those in our deviation plots.

**Comment by jpotoff** [comment]
@ramess101 That's unfortunate that we missed their cyclohexane model.  I should have known better as I've seen that paper before, but my brain was a bit foggy last semester.  Still, the paper is about a method and an optimization process, so this isn't a deal breaker.  We include the TAMie data, and use it as validation of the method.  We start from TraPPE, optimize parameters, and get to TAMie (about).  I'm wondering what our data say about the deviation between the predictions of the TAMie parameters and experiment.  Is it the same is what is published, or do we see slightly different results?  That might be a good comparison to include in supporting information if they match.  If they don't...Maybe still throw it in supporting information.  If we don't have it in the heat map, it would only take a day or two to run it (and I think we should).  

BTW, I do have a concern about this being an apples to apples comparison, though.  If we subjected the exp-6 and/or the TAMie potential to the same thoroughness of calculation (20 independent runs), I would expect to see some shifts in the data.

**Comment by ramess101** [comment]
@jpotoff 

> Still, the paper is about a method and an optimization process, so this isn't a deal breaker.  We start from TraPPE, optimize parameters, and get to TAMie (about).

I agree. It certainly doesn't minimize the influence of our methods, but the culmination with MiPPE cyclohexane parameters losses a bit of its punch. I need to modify the story we are telling a bit in the Case Study section too. 

> I'm wondering what our data say about the deviation between the predictions of the TAMie parameters and experiment. Is it the same is what is published, or do we see slightly different results? That might be a good comparison to include in supporting information if they match. If they don't...Maybe still throw it in supporting information. If we don't have it in the heat map, it would only take a day or two to run it (and I think we should).

I will let you know once I have these results. Even if our results differ slightly from the literature TAMie values, this could be attributed to finite-size effects (considering their alternative DeltaHv approach) and partially the difference in torsion potential. But I am pretty confident they will agree within the combined uncertainties.

> BTW, I do have a concern about this being an apples to apples comparison, though. If we subjected the exp-6 and/or the TAMie potential to the same thoroughness of calculation (20 independent runs), I would expect to see some shifts in the data.

The main reason of including Exp-6 and TAMie results was to meet the JCED requirement that we compare our results with preexisting literature values.

**Comment by mrshirts** [comment]
> The main reason of including Exp-6 and TAMie results was to meet the JCED requirement that we compare our results with preexisting literature values.

Right, so hopefully we can do this, and show that  the method works . . . 

> BTW, I do have a concern about this being an apples to apples comparison, though. If we subjected the exp-6 and/or the TAMie potential to the same thoroughness of calculation (20 independent runs), I would expect to see some shifts in the data.

And that could be pointed out.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

I have not finished modifying the manuscript, but this is what I am planning:

![image](https://user-images.githubusercontent.com/23408516/53752634-57169a80-3e6c-11e9-9769-177320631119.png)

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

This is what the deviation plots look like with TAMie included (grey circles):

![image](https://user-images.githubusercontent.com/23408516/53752887-1bc89b80-3e6d-11e9-8d8e-4a629cc3da2b.png)

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

I am going to include this figure in the SI. Clearly the MiPPE (and to a lesser extent the first iteration of MiPPE) results agree very nicely with the TAMie reported values. Interesting that rhol is the largest deviation. The purported finite-size effects for DeltaHv at low T are nonexistent.

![image](https://user-images.githubusercontent.com/23408516/53753498-a65dca80-3e6e-11e9-8d1e-402000c6d2b1.png)

**Comment by ramess101** [comment]
@jpotoff @msoroush 

>  I'm wondering what our data say about the deviation between the predictions of the TAMie parameters and experiment.  Is it the same is what is published, or do we see slightly different results?  That might be a good comparison to include in supporting information if they match.  If they don't...Maybe still throw it in supporting information.  If we don't have it in the heat map, it would only take a day or two to run it (and I think we should).

OK, after doing some digging we did compute phase properties for the TAMie parameters in our refined second iteration heat map scan, at least if we round the parameters to within 3 digits of precision (which is not uncommon). Specifically, TAMie reports epsilon of 69.568 K and sigma of 3.8967 Angstrom while we predicted values for 69.5 or 69.6 K and 3.895 or 3.900 Angstrom. The values shown here as "TAMie, This work" are for 69.6 K and 3.9 Angstrom. 

I think this is good enough to show that our results are consistent. I could run for the EXACT same epsilon and sigma, but I think we are well within the noise at this point. Also, I don't have access to my system at NIST where I ran these calculations originally. 

We could always run a full set of GCMC simulations for TAMie (20 replicates, etc.). But I think that is overkill. Any objections?

![image](https://user-images.githubusercontent.com/23408516/53764652-1ed28480-3e8b-11e9-87a3-b6081e61f235.png)


![image](https://user-images.githubusercontent.com/23408516/53764641-10846880-3e8b-11e9-8e52-47a5a671e577.png)

**Comment by ramess101** [comment]
@jpotoff @msoroush 

But if you and Mohammad want to run 20 replicates with TAMie, I will certainly include those results instead of mine. My GCMC-MBAR results are admittedly not the best values to report because they are estimated from a single set of simulations performed with the first iteration theta parameter set. Mohammad would probably need to play around with the best mu values for TAMie. And he might want to run with 3.5 nm (?)

**Comment by msoroush** [comment]
@ramess101  I can run the TAMie by tomorrow. What is the exact parameter you want me to run? And what box size?

**Comment by ramess101** [comment]
@msoroush 

OK, perfect! Let's use the exact values that TAMie reported:

![image](https://user-images.githubusercontent.com/23408516/53767038-c357c500-3e91-11e9-8cfa-4f01cea6bab6.png)

Weidler et al. reported using box sizes between 40,000 A^3 (for propanal) and 81,000 A^3 (for octanal), so I would say that we should use the 35 Angstrom box length instead of 30 Angstrom box length (since this is only 27000 A^3).

**Comment by msoroush** [comment]
@ramess101 I also used their reported torsion

**Comment by ramess101** [comment]
@msoroush 

Good point. So you have their modified torsion potential coded in GOMC?

**Comment by msoroush** [comment]
@ramess101 I used the cos series but I converted it using ``` C0 = C0 - (C1 + C2 + C3) ```

**Comment by ramess101** [comment]
@msoroush 

Perfect. Interesting that they (both TAMie and TraPPE) didn't just correct the C0 term in their table instead of making it look like it is a completely different equation.

**Comment by msoroush** [comment]
@ramess101 have you tried to plot the torsion energy for the equation and parameter used in TAMie? Is the minimum correct?

**Comment by msoroush** [comment]
@ramess101 @jpotoff  C1 term must be negative. Otherwise we face the same problem. I am sampling boat configuration and no crown! You might put back your old statement about wrong torsion value.

**Comment by ramess101** [comment]
@msoroush 

I have not, I just assumed that because they changed the equation that it fixed the problem. I will go ahead and put back in the old statement unless you get this to work.

So should we simulate TAMie with the same torsion that we used for MiPPE?

**Comment by msoroush** [comment]
This is TAMie torsion with original parameter:
![tamie_org](https://user-images.githubusercontent.com/13317444/53830907-7c310900-3f51-11e9-8984-c369de630d20.png)

This is TAMie torsion with modified parameter (-C1 term):
![tamie_modified](https://user-images.githubusercontent.com/13317444/53830939-8eab4280-3f51-11e9-9727-174cb79dd4fb.png)

This is alkane torsion parameter:
![alkane_torsion](https://user-images.githubusercontent.com/13317444/53830965-9c60c800-3f51-11e9-9e28-444052baa853.png)

**Comment by msoroush** [comment]
@ramess101 @jpotoff 

Is that okay if I use modified torsion of TAMie?

**Comment by ramess101** [comment]
@msoroush @jpotoff 

I think whatever you used for MiPPE should be ok for TAMie. The TAMie equation and parameters are supposed to be the same as TraPPE. And so if changing the equation still requires making c1 negative, I think it would be simpler to just use what we did for MiPPE previously (use alkane).

**Comment by msoroush** [comment]
@ramess101 @jpotoff  Okay. I will use alkane torsion parameters.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

Included this statement that MiPPE and TAMie are very similar. Also wanted to make it clear that we are not purporting that MiPPE is better than TAMie since TAMie used a slightly different objective function (relative squared error for only rhol and Psat) and experimental data set (DIPPR):

![image](https://user-images.githubusercontent.com/23408516/53908519-bf03e700-400c-11e9-956e-1c2b87402d44.png)

**Comment by ramess101** [comment]
@msoroush 

Do you expect the TAMie results by tomorrow? I think that is the last thing that I need before I can resubmit. Thanks

**Comment by msoroush** [comment]
@ramess101 I am still working on fixing the chemical potential. I will have results for one replica. I will set it to run for 20 replica, by tonight but not sure when they will be finished. When is the deadline to resubmit it?

**Comment by ramess101** [comment]
@msoroush 

That's fine. The deadline is next Monday but I just didn't want to cut it too close, to give us time in case the results don't come out as we expect.

**Comment by msoroush** [comment]
@ramess101 Do you want the single replica results? Wrong torsion parameters messed up my schedule :D

**Comment by ramess101** [comment]
@msoroush 

Haha, those torsions are going to be the death of us someday...

Sure, pass the single replica results along when you have a chance. That way I can make sure they look reasonable compared to Weidler et al.

**Comment by msoroush** [comment]
@ramess101 here is the data for single replica. Simulations are not finished, I had to kill them to start other simulations. So, the quality might not be as good as we want, specially at lower temperature. 
[vle.zip](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2938846/vle.zip)

**Comment by ramess101** [comment]
@msoroush @jpotoff 

 I am a little bit concerned about our TAMie replicates. I know this was just a single (short) replicate, but the deviation in Psat and rhov is way greater than what Weidler reports. 

![image](https://user-images.githubusercontent.com/23408516/53969731-89b3d380-40b6-11e9-94d3-8d54c3714c97.png)

**Comment by ramess101** [comment]
@jpotoff @msoroush 

OK, I looked into this a little more. And I think your trends for TAMie are actually correct. The figures that I plotted previously with my own GCMC-MBAR predicted curves for TAMie were not for the correct sigma. I plotted results for eps = 69.6 K (which is the closest I have to TAMie of 69.568), but I was using MiPPE sigma (0.3902) instead of TAMie sigma (0.3895 or 0.3900 are the two closest I have to 0.38967).

Here are my results for sigma of 0.3895:

![image](https://user-images.githubusercontent.com/23408516/53970202-69384900-40b7-11e9-96bb-d161cc73c657.png)

Here are my results for sigma of 0.3900:

![image](https://user-images.githubusercontent.com/23408516/53970167-5756a600-40b7-11e9-95b7-de4106513873.png)

If you mentally interpolate the Psat and rhov errors for the two sigma values, you will see that my TAMie results actually do agree with your TAMie results. In other words, the Weidler et al. Psat and rhovap values are quite different. Could this be related to the torsions? I think we will need to anticipate our TAMie results not matching Weidler et al. and provide some explanation for this in either the review or the manuscript.

**Comment by ramess101** [comment]
@jpotoff @msoroush 

The TraPPE results provide additional evidence that the torsions might be to blame for this discrepancy. Our Psat results agree with those of Yiannourokou (who also used the alkane torsion potential) but disagree with Keasler (who purportedly used the TraPPE cyclic potential). We don't really agree with either for rhovap, but I think this might explain the few percent shift in Psat for TAMie.

![image](https://user-images.githubusercontent.com/23408516/53971138-85d58080-40b9-11e9-8890-af22dcd8fb53.png)

**Comment by ramess101** [comment]
@msoroush @jpotoff 

So after these 20 replicates finish, I think we might want to try simulating TAMie using the TraPPE cyclic potential (but with a negative c1 value).

**Comment by msoroush** [comment]
@ramess101 for MIPPE data, are we using 30 or 35 A data?

**Comment by ramess101** [comment]
@msoroush 

Good point. These MiPPE values were for 30 just because that is what we used in the optimization. But I could use 35 instead to be consistent with our TAMie values. The results do not differ by several percent though (except at high T) so that doesn't really explain the discrepancy.

**Comment by msoroush** [comment]
@ramess101 I am looking at my TraPPE results and I see the same discrepancy in vapor density. I think, the best way to sort it out is to run NPT simulation in gas phase using MD code, for each torsion model (alkane torsion and TraPPE torsion) and compare the predicted densities. 

I am not sure how with wrong torsion parameter, they can get these results. In GOMC if I used the wrong torsion (TraPPE torsion), my liquid density will be much higher than what they reported.

**Comment by ramess101** [comment]
@msoroush 

Yeah, I don't know how they could get their results with the reported torsions. But maybe if it really is a sign error for C1, the results would shift slightly?

**Comment by msoroush** [comment]
@ramess101 Here are the results for alkane torsion and modified torsion (-C1 term). I dont see any changes in the data.

[cyclohexane.tar.gz](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2942481/cyclohexane.tar.gz)

**Comment by ramess101** [comment]
@msoroush 

Strange. So if it is not the torsions, then maybe there is some subtle difference in the methods, codes, etc. 

What about the cutoff length? Are we using 10 A or 14 A? TAMie and TraPPE both use 14 A cutoff.

**Comment by msoroush** [comment]
@ramess101 @jpotoff 

The problem is that vapor density results for Keasler does not match with Yiannourakou, either. 

Using rcut 10 vs 14 would not affect the results as long as we use Long Range Correction. We can look at the branched alkane results and compare it with TraPPE results.

**Comment by ramess101** [comment]
@msoroush @jpotoff 

True, but Keasler used GEMC and didn't report uncertainties, so I am more interested about the TAMie comparison. I expect the TAMie Weidler results are quite reliable and were computed with a similar GCMC approach (although they did use a different method for computing pressure as mentioned in Comment #29). 

I agree that with long-range corrections you typically get indistinguishable values for rcut of 10 and 14, but in some cases the assumption of g=1 at rcut=10 may not be valid (rings?).

**Comment by ramess101** [comment]
@msoroush @jpotoff 

My vote would be that we repeat the single replicate (at least) with rcut of 14 for TAMie, just to be safe. The box length is 35 A, so rcut = 14 should not be a problem there.

**Comment by msoroush** [comment]
> @msoroush @jpotoff
> 
> My vote would be that we repeat the single replicate (at least) with rcut of 14 for TAMie, just to be safe. The box length is 35 A, so rcut = 14 should not be a problem there.

Okay, I will set it up.

**Comment by ramess101** [comment]
@msoroush 

Thanks! Sorry for being so demanding... I just want to make sure we leave no stone unturned

**Comment by msoroush** [comment]
@ramess101 
Just to be clear, you want me to repeat the cyclohexane simulation for TAMie model using rcut 14 A and 35 A cubic box?

**Comment by ramess101** [comment]
@msoroush 

Yeah. And using the alkane torsion is fine too.

**Comment by msoroush** [comment]
@ramess101 Do you have time to setup GROMACS to run NPT simulation for liquid and vapor phase?

**Comment by ramess101** [comment]
@msoroush 

Sorry, I actually do not. I have zero computational resources (not set-up yet at NREL and no access at NIST). So I am kind of useless and I definitely can't get anything done by Monday. 👎

**Comment by msoroush** [comment]
@ramess101 You ran the initial simulation for TraPPE in GOMC? Would you please plot the densities predicted by GOMC, Keasler, and  Yiannourakou?

**Comment by ramess101** [comment]
@msoroush 

Yeah, that is what is shown in this figure:

![image](https://user-images.githubusercontent.com/23408516/54046431-84778700-4191-11e9-93b3-9d9396638b02.png)

**Comment by msoroush** [comment]
This is the my vapor density data for TraPPE. This difference is not visible when you plot the density.
![vap](https://user-images.githubusercontent.com/13317444/54048431-c6132c80-41a7-11e9-940b-c8a461fec9fe.png)

**Comment by msoroush** [comment]
This data for one replica must have better quality.Can you use this data for TAMie to compare with their data?

[VLE_10A_TAMie.tar.gz](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2947049/VLE_10A_TAMie.tar.gz)

**Comment by msoroush** [comment]
@ramess101 

If you are looking for Keasler uncertainties in VLE data, here is the [link](http://chem-siepmann.oit.umn.edu/siepmann/trappe/vlccdetail.html) for it.

**Comment by ramess101** [comment]
@msoroush 

Where did you get the "GOMC" values for the figure you plotted above? Was that from recent simulations? I think if you convert that to a percent deviation plot you are going to see a similar discrepancy because the green points appear to be slightly shifted (on a log plot).

**Comment by ramess101** [comment]
@msoroush 

Thanks for directing me to Keasler uncertainties. They weren't reported in the original manuscript, but I didn't think to look on the website.

**Comment by msoroush** [comment]
@ramess101  I ran it before, when I was trying to find the problem with torsion.

**Comment by ramess101** [comment]
@msoroush 

OK, but do you agree that there is also a small shift in vapor density? If you have the data, I could throw it on a deviation plot to see if it really does deviate slightly

**Comment by msoroush** [comment]
@ramess101 I totally agreeing with you. I can see the deviation, but when you calculate the error, you can see it more clearly. Lets see how TAMie results looks like. One other thing I found out, in their paper, they mentioned that they used tabulated parameter for TAMie and used reservoir method for molecule insertion/deletion.

**Comment by ramess101** [comment]
@msoroush 

Cool, just wanted to make sure we were on the same page. Here are the TAMie results with the updated values you uploaded:

![image](https://user-images.githubusercontent.com/23408516/54053311-12f50400-41a4-11e9-8d0a-c2e683690732.png)

**Comment by ramess101** [comment]
@msoroush 

On Towhee and GROMACS I have always obtained consistent results for tabulated or native LJ potentials, but I guess it could impact your results if the spacing is not refined.

**Comment by ramess101** [comment]
@msoroush 

Here I have replotted the Keasler comparison with the uncertainties:

![image](https://user-images.githubusercontent.com/23408516/54054591-dcb98380-41a7-11e9-9435-8cc140f543cb.png)

**Comment by msoroush** [comment]
> @msoroush
> 
> Here I have replotted the Keasler comparison with the uncertainties:
> 
> ![image](https://user-images.githubusercontent.com/23408516/54054591-dcb98380-41a7-11e9-9435-8cc140f543cb.png)

Our data are between Keasler and Yiann, data :D

**Comment by ramess101** [comment]
@msoroush 

It is curious that our rhol and DeltaHv are much lower for TraPPE, while our rhol is higher and DeltaHv lower for TAMie

**Comment by ramess101** [comment]
@msoroush 

So do we anticipate having results by Monday for TAMie with 10 A cut-off and maybe 14 A?

**Comment by msoroush** [comment]
@ramess101 Hopefully they both will be finished by tomorrow. As soon as I get the results. I will send it to you.

**Comment by ramess101** [comment]
@msoroush 

Alright sounds good. Thanks again!

**Comment by ramess101** [comment]
@msoroush @jpotoff 

I think we need to anticipate that our TAMie results are going to have slight (but irreconcilable) deviations from Weidler et al. How do we want to address this in the manuscript/review? The deviations in DeltaHv are particularly concerning since the reviewer claims that our approach will suffer from finite-size effects (although our comparison for system sizes didn't seem to suggest that was true).

**Comment by msoroush** [comment]
@ramess101 we tried their method to calculate the dHv. Results where changed slightly for molecules with intrabonded energy, which would result in change in the error value. Maybe we can calculate dHV with their method, and plot them together to see if the dHv for cyclohexane changes or not.

**Comment by ramess101** [comment]
@msoroush 

OK, that might be the cause of the discrepancy. We assume that intrabonded is the same in liquid and vapor phase right? This certainly could contribute around 1% to dHv. Do you have those dHv values readily available?

**Comment by jpotoff** [comment]
The TAMie method for calculating heat of vaporization appears to be more resistant to finite size effects in the near critical region.  It doesn't seem to offer better performance with respect to finite size effects at lower temperatures.  We did see a small shift in the calculated error going from our method to the TAMie way of doing things.  It might be due to the bonded interactions, but we don't have conclusive evidence, yet.  @msoroush Could you post Niloo's figures for methane and butane to illustrate the difference?  The only way to really tell would be to compare to GEMC simulations where we calculate dHv using bonded and non-bonded interactions.  Unfortunately, the uncertainty in those data is a lot larger than the GCMC results...

**Comment by ramess101** [comment]
@jpotoff @msoroush 

OK, so the difference at low T is not due to finite size effects. But it could be caused by how dHv is being computed, namely, their approach makes no assumption about intramolecular energies cancelling between the two phases.

Yeah, the fact that there might be some discrepancy between GEMC and GCMC is not totally unexpected. I don't think we need to run GEMC simulations for this study, although it could be worth looking into.

For the purpose of this review, we just need to show that the deviation in TAMie is: 1) Not due to finite-size effects 2) Caused by different analysis methods, where our approach assumes intramolecular energies are the same in vapor and liquid. Right?

**Comment by msoroush** [comment]
Butane:
![butane-averages](https://user-images.githubusercontent.com/13317444/54060637-7b0b1080-41cb-11e9-8614-97a3e9404bf5.png)

Methane:
![methane-averages](https://user-images.githubusercontent.com/13317444/54060669-96761b80-41cb-11e9-96aa-106418e94a91.png)

**Comment by ramess101** [comment]
@msoroush

So the left panels use our method and the right panels is TAMie method?
Discrepancy in methods appear larger for butane, supporting the
intramolecular theory

**Comment by msoroush** [comment]
@ramess101 

Here are the results for rcut10 and 14 for TAMie
[rcut10.zip](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2949104/rcut10.zip)
[rcut14.zip](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2949105/rcut14.zip)

**Comment by ramess101** [comment]
@msoroush 

The rcut14 looked marginally better, and I think we should include the rcut14 results. The rcut14.zip folder was missing the Average_Results_m1.dat file. Could you re-zip rcut14 with this file and re-upload it?

**Comment by msoroush** [comment]
@ramess101 Here are the average results
[rcut14.zip](https://github.com/ramess101/JCED_FOMMS_Manuscript/files/2952649/rcut14.zip)

**Comment by ramess101** [comment]
@msoroush @jpotoff 

OK, here are the comparisons for our TAMie simulations:

rcut = 10 A

![image](https://user-images.githubusercontent.com/23408516/54133569-efb49980-43db-11e9-99d6-4b40478d31fc.png)

rcut = 14 A

![image](https://user-images.githubusercontent.com/23408516/54133557-e9262200-43db-11e9-9397-0091f45bf048.png)

**Comment by ramess101** [comment]
@msoroush @jpotoff 

rcut = 14 A is slightly better, but still a clear discrepancy.

Is this unexpected? Of course our DeltaHv analysis is different, but would you expect the disagreement for rholiq and Psat? Could the difference in Psat be related to the different approaches for computing absolute pressures (recall Issue #29)? But this shouldn't affect rholiq, right? Because equating the pressures in the two phases should still be the same.

**Comment by ramess101** [comment]
@msoroush @jpotoff 

In case you didn't know, the response to the reviewer comments is due today. I think this is really the only issue that we still need to resolve. Please let me know if you have any thoughts for how we can adequately explain the discrepancy between our TAMie results.

**Comment by ramess101** [comment]
@msoroush 

Would you be able to compute DeltaHv for our TAMie simulations (14 A) using the TAMie methodology? Including those values might be helpful.

**Comment by jpotoff** [comment]
The agreement for rho_liquid and dHv, is reasonably decent; at least we have overlap of the error bars.  It's the vapor density and vapor pressure that are way off.  Given that the vapor densities are different, the difference in the vapor pressure is unlikely to be due to the different techniques used to get the reference state.  Without seeing their simulation and histogram analysis code, it is impossible to explain  why the results are different with any certainty.  

Looking at Gross's data, if those were our data, I would concerned about the large error bars, especially near the critical point.  That seems very odd.

**Comment by ramess101** [comment]
@jpotoff 

Good point about the vapor densities. I will plot up the Zvap really quick to see what those look like.

**Comment by ramess101** [comment]
@jpotoff 

However, the rholiq deviations are actually not within the error bars at most temperatures (compare "TAMie, This Work" with "TAMie, Weidler et al." DeltaHv are closer though

**Comment by jpotoff** [comment]
> However, the rholiq deviations are actually not within the error bars at most temperatures (compare "TAMie, This Work" with "TAMie, Weidler et al." DeltaHv are closer though

Right.  I was incorrectly comparing our results and TAMie instead of comparing TAMie (us) with TAMie (published).  We may just have to throw our hands up on this one since the TAMie calculations are done with a non-open source simulation engine.

**Comment by ramess101** [comment]
@jpotoff 

Yeah, currently I just try to pass this off as relatively small deviations for rhol and DeltaHv (less than 0.5%)

**Comment by ramess101** [comment]
@msoroush @jpotoff 

Here is a comparison for the Zvap values:

![image](https://user-images.githubusercontent.com/23408516/54148298-95c2cc80-43f9-11e9-9c9f-994e734db507.png)

I think this shows that the deviation in Psat is due to the deviations in rhovap because we get the same Z, right?

**Comment by jpotoff** [comment]
I think my brain just exploded.  Pvap  and rho_vap are different between our results and Gross, but we get cancellation of errors to give an exact match for Z=PV_gas/RT?  I'm having a hard time reconciling these results.

**Comment by ramess101** [comment]
@jpotoff 

My mindset was that this demonstrates that the ideal gas is not to blame for the discrepancy in Psat. Because we still get Z converging to 1, the pressure calculations are consistent with the densities, right?

**Comment by msoroush** [comment]
@ramess101 What if we plot error in Z? I think we will see some deviation.

**Comment by ramess101** [comment]
@msoroush 

Yeah, but we can tell that the error bars do overlap. Of course, the Z error bars might be too large since I assumed independence between rhovap and Psat...let me fix that first

**Comment by ramess101** [comment]
@msoroush @jpotoff 

![image](https://user-images.githubusercontent.com/23408516/54153512-64e89480-4405-11e9-9083-e44ae5f68210.png)

**Comment by jpotoff** [comment]
That resolves the comment about reference states.  

Still having a hard time resolving that rho_vap and Pvap don't agree with us.  I guess we have to accept that we aren't going to be able to answer that question conclusively.  Then again, we weren't asked to do that by the reviewers. If we stick with just Z and put the plot in the supporting info, or in the response to the reviewers, that should be sufficient.

**Comment by ramess101** [comment]
@jpotoff 

Agreed.

**Comment by ramess101** [comment]
@msoroush @jpotoff 

Any suggestions or concerns for this portion of the response:

![image](https://user-images.githubusercontent.com/23408516/54160517-61f5a000-4415-11e9-93c1-851719831f1a.png)

![image](https://user-images.githubusercontent.com/23408516/54160581-8ea9b780-4415-11e9-9b46-7313af8f88e7.png)


![image](https://user-images.githubusercontent.com/23408516/54160555-7e91d800-4415-11e9-9620-9a0e509a3f80.png)

**Comment by jpotoff** [comment]
This looks good. We've gone above and beyond addressing the reviewer comments.

**Comment by ramess101** [comment]
@jpotoff

Thanks, agreed!

**Comment by ramess101** [comment]
@msoroush

Do you have the state points (chemical potentials) that you used to
simulate TAMie (just for 3.5 nm box and 1.4 nm cutoff simulations)? Just
needed for the SI

**Comment by msoroush** [comment]
@ramess101 Here are the state point for TAMie
```
RUNS
===============================================
-   #    T(K)      Chem. Pot   CBMC_LJ_1st CBMC_LJ_nth
-----------------------------------------------------------
R   1    450        -4358       8          4
R   2    510        -4358       8          4
R   3    550        -4358       8          4
R   4    510        -4165       8          4
R   5    470        -4036       8          4
R   6    430        -3923       16        10
R   7    400        -3852       16        10
R   8    370        -3793       16        10
R   9    350        -3762       16        10
```

**Comment by ramess101** [comment]
@msoroush

Perfect
